jujugui/static/gui/src/test/test_env_go.js

```diff
@@ -710,9 +745,79 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
               DefaultSeries: 'trusty',
               Name: 'admin',
               ProviderType: 'lxd',
+              UUID: '5bea955d-7a43-47d3-89dd-b02c923e',
               ServerUUID: '5bea955d-7a43-47d3-89dd',
-              UUID: '5bea955d-7a43-47d3-89dd-b02c923e2447',
-              'owner-tag': 'user-admin@local'
+              Life: 'alive',
+              OwnerTag: 'user-admin@local'
+            }
+          }]
+        }
+      });
+    });
+
+    it('retrieves model info for multiple models', function(done) {
+      // Perform the request.
+      var tag1 = 'model-5bea955d-7a43-47d3-89dd-tag1';
+      var tag2 = 'model-5bea955d-7a43-47d3-89dd-tag2';
+      env.modelInfo([tag1, tag2], function(data) {
+        assert.strictEqual(data.err, undefined);
+        assert.strictEqual(data.models.length, 2);
+        var result1 = data.models[0];
+        assert.strictEqual(result1.tag, tag1);
```

Comment:
Yes, I reused the same pattern used elsewhere and didn't change it with the consideration that deepEquals, when it fails, produces a meaningless output.